src/Pages/Portal/Login/Login.tsx

```diff
@@ -0,0 +1,223 @@
+import React, { SetStateAction, useState } from 'react'
+import {
+    Button,
+    Input,
+    InputGroup,
+    Box,
+    Flex,
+    Text,
+    Center,
+    FormControl,
+    Spacer,
+    HStack,
+    InputLeftElement,
+} from '@chakra-ui/react'
+import { FaFacebook, FaGithub } from 'react-icons/fa'
+import { EmailIcon, LockIcon } from '@chakra-ui/icons'
+import { FcGoogle } from 'react-icons/fc'
+import {
+    BrowserRouter as Router,
+    Routes as Switch,
+    Route,
+    Link as RouteLink,
+} from 'react-router-dom'
+import '../../../Backend/index'
+import { useForm, SubmitHandler } from 'react-hook-form'
+import { useDisclosure } from '@chakra-ui/react'
+import signIn from '../../../Backend/Routes/signIn'
+
+interface LoginProps {
+    updateIsLogin: React.Dispatch<SetStateAction<boolean>>
+}
+
+type Inputs = {
+    email: string
+    password: string
+}
+
+function Login(props: LoginProps) {
+    const {
+        register,
+        handleSubmit,
+        watch,
+        formState: { errors },
+    } = useForm<Inputs>()
+
+    const onSubmit: SubmitHandler<Inputs> = async (data) => {
+        const success = await signIn(data.email, data.password)
+        if (success) {
+            console.log('yay') // go to application page
+        } else {
+            console.log("Couldn\'t log in")
+        }
+    }
+
+    return (
+        <Flex height="100vh">
+            <Flex flex='2.5' bg='#F8F8F8' alignItems='start' justifyContent="center" pt="5%">
+                <Flex direction="column" width="60%" alignItems="center" justifyContent="start">
+                    <Text
+                        color="#211E61"
+                        fontSize="5xl"
+                        fontFamily="P052"
```

Comment:
Define global font for CHakra to be this, not everywhere. Use font theme style, not defining each style individually. Use theme colors instead of hex. No percentage spacing